client/app.js

```diff
@@ -64,7 +64,7 @@ app.config(['$stateProvider', '$urlRouterProvider',
 
       })
       .state('graph', {
-        // url: '/graph', // optional
+        url: '/graph', // optional
```

Comment:
Note: No URLs have been specified for any other states yet. We should change the graph state to be consistent with the others.